### ISSUE TITLE
snooze delay slider UI improvement suggestion

### ISSUE THREAD

**Issue by petep0p** [nproth/pin#37]
this should really be under the closed issue #35 
you mentioned the snooze delay slider might have issues with discoverability, and i had 2 thoughts on how to make it more discoverable.
1) make the semicircle line terminate in a ball rather than just a plain line. this would make it more obvious that it's a slider
2) make it change position when a different delay time is chosen by tapping. in other words, it would display as almost turned all the way down when you tap to 0:30 and almost all the way turned up when you tap to 24:00. this would also make it more obvious it can change.

these two things are not mutually exclusive. anyway, just a suggestion that popped into my head that i thought I should share. do with it what you will.

**Comment by nproth** [comment]
Yes, good ideas.
Regarding *2.*, it would be finicky to hit 30 minutes or less on a 24 hour scale. Animating the slider on tap or when shown could serve as indication instead.

**Comment by petep0p** [comment]
yeah that would probably work better. i don't think it would serve any purpose to keep it exactly on scale, but having it move to indicate that it can be moved would certainly get the point across.
